roles/rsyslog/tasks/inputs/remote/main.yml

```diff
@@ -9,15 +9,26 @@
 
 - name: Create basics input configuration files in /etc/rsyslog.d
   vars:
-    __rsyslog_input_decl: ""
-    __udp_input_calls: "# Log messages from remote hosts over UDP\ninput(name=\"{{ remote_item.name }}\" type=\"imudp\" port=\"{{ remote_item.udp_port | d(__rsyslog_default_port) }}\")\n"
-    __ptcp_input_calls: "# Log messages from remote hosts over plain TCP\ninput(name=\"{{ remote_item.name }}\" type=\"imptcp\" port=\"{{ remote_item.tcp_port | d(__rsyslog_default_port) }}\")\n"
-    __tls_input_calls: "# Log messages from remote hosts over TLS\ninput(name=\"{{ remote_item.name }}\" type=\"imtcp\" port=\"{{ remote_item.tcp_tls_port | d(__rsyslog_default_tls_port) }}\")\n"
     __rsyslog_input_name: "{{ remote_item.name }}"
     __rsyslog_input_type: "{{ remote_item.type }}"
     __rsyslog_packages: []
-    __rsyslog_rules: [ { "name": "input-remote-{{ remote_item.name }}", "type": "input", "weight": "11",
-                         "sections": [ { "options": "{{ __udp_input_calls if (remote_item.udp_port | d() ) else ( __ptcp_input_calls if remote_item.tcp_port | d() else __tls_input_calls ) }}{{ lookup('template', 'input_template.j2') }}" } ] } ]
```

Comment:
@nhosoi You say "__rsyslog_rules value from a string to the dictionary" but __rsyslog_rules was a dictionary before and it is a dictionary after the change.

Reply:
@pcahyna, indeed :p. I'll fix the comment to `modifying the __rsyslog_rules value to YAML friendly manner.`